package-support-policy.md

```diff
@@ -0,0 +1,67 @@
+
+## Reporting issues
+
+If you find an issue in an assembly that is hosted in this repo, please report it in our central repo: https://github.com/dotnet/runtime/issues/new/choose.
+
+## Should I consume this package? Which version should I consume?
+
+Please refer to the table below to:
+
+- Learn if a package is still supported and in which target frameworks.
+- Learn what action you should take if you're actively consuming one of these packages.
+
+| Package                                 | Still supported? | Reasons                                         | Actions                                |
+|-----------------------------------------|------------------|-------------------------------------------------|----------------------------------------|
+| Microsoft.Bcl.HashCode                  | Yes              | Provides the implementation of [`HashCode`](https://learn.microsoft.com/en-us/dotnet/api/system.hashcode) for .NET Framework. | <ul><li>.NET Framework 4.6.2+: Update to latest version of the package.</li><li>.NET 6+: No need to reference the package. The type is part of the shared framework.</li></ul> |
+| Microsoft.CSharp                        | Limited          | Contains an implementation but it is unused in all the currently supported frameworks. | Only reference the package if you're targeting .NET Standard 2.0 and you need to use the Microsoft.CSharp. The implementation found in the package won't be used, but will make sure the namespace is found. The in-box implementation will be used instead. This package won't take any fixes. |
+| Microsoft.IO.Redist                     | Yes              | Intended for internal use only, not for general use. | Avoid referencing the package. Use the equivalent System.IO APIs instead. |
+| System.Buffers                          | Yes              | Provides the implementation of [`ArrayPool<T>`](https://learn.microsoft.com/en-us/dotnet/api/system.buffers.arraypool-1) for .NET Framework. | <ul><li>.NET Framework 4.6.2+: Update to the latest version of the package.</li><li>.NET 6+: No need to reference the package. The types are part of the shared framework.</li></ul> |
+| System.Data.SqlClient                   | Limited          | [Supported until .NET 8 goes EOL.](https://techcommunity.microsoft.com/blog/sqlserver/announcement-system-data-sqlclient-package-is-now-deprecated/4227205) | <ul><li> .NET Framework 4.6.2+: The .NET Framework implementation is simply an in-box façade. No need to reference the package.</li><li> .NET Standard 2.0+, .NET 6.0 and .NET 7.0: System.Data.SqlClient is no longer supported. Migrate to Microsoft.Data.SqlClient.</li><li>.NET 8: System.Data.SqlClient is obsolete. Consider migrating to Microsoft.Data.SqlClient.</li><li>.NET 9+: System.Data.SqlClient is no longer supported. Migrate to Microsoft.Data.SqlClient.</li></ul> |
```

Comment:
Will users know what "is simply an in-box façade" means? Would "is forwarded to the system-installed .NET Framework" be clearer?

Reply:
Good point. I rewrote the sentence to include your suggestion.